io.asgardeo.java.oidc.sdk/src/main/java/io/asgardeo/java/oidc/sdk/HTTPSessionBasedOIDCProcessor.java

```diff
@@ -63,6 +65,11 @@ public void sendForLogin(HttpServletRequest request, HttpServletResponse respons
 
         HttpSession session = request.getSession();
         RequestContext requestContext = defaultOIDCManager.sendForLogin(request, response);
+        if (request.getRequestURI() != null) {
+            String[] uri = request.getRequestURI().toString().split("/");
+            String redirectPageURI = String.join("/", Arrays.copyOfRange(uri, 2, uri.length));
+            requestContext.setParameter(SSOAgentConstants.REDIRECT_URI_KEY, redirectPageURI);
+        }
```

Comment:
`Arrays.copyOfRange(uri, 2, uri.length)`  wouldn't this be erroneous if the app context path has a URL like `abc.com/t/xyz/apps/my-app` ?

Reply:
better to get the relative URL for the servlet